www/deno.json

```diff
@@ -8,7 +8,7 @@
     "$ga4": "https://raw.githubusercontent.com/denoland/ga4/main/mod.ts",
     "@fresh/plugin-tailwind": "../plugin-tailwindcss/src/mod.ts",
     "@luca/esbuild-deno-loader": "jsr:@luca/esbuild-deno-loader@^0.11.0",
-    "@preact/signals": "npm:@preact/signals@^1.3.0",
+    "@preact/signals": "npm:@preact/signals@^2.0.4",
```

Comment:
Is it not possible to remove this dependency from here instead? Will that not use the same version as the root `deno.json` instead via `workspaces`?